.eslintrc

```diff
@@ -3,13 +3,14 @@
 extends:
   - standard
   - standard-react
+  - prettier
```

Comment:
do we need `standard` if we have `prettier`? I don't know if some/all rules are redundant

Reply:
I will remove standard and make some tests. But I think you're right.